### ISSUE TITLE
Data is updated once every 4 iterations during PhaseB

### ISSUE THREAD

**Issue by lavoiems** [Joshua-Ren/Neural_Iterated_Learning#3]
See: https://github.com/Joshua-Ren/Neural_Iterated_Learning/blob/master/train.py#L222.

It is common to update the data every iterations. Since I don't remember seeing it being discussed in the paper, I am wondering if this is intended. If so, what is the rational behind not updating the data every iteration?

Thank you.

**Comment by Joshua-Ren** [comment]
Hi Sam,

Thanks very much for your question. I am sorry for not mentioning this
small trick in the appendix (oh actually I delete this in the final
version). This trick seems can stabilize the training during the ablation
study, i.e., only Alice (or Bob) is updated when the learning rate is
small. (It can let the agent learn the mapping better.) But I think you can
safely delete this line without harming the performance. Because later I
found that the training can still converge as long as the learning rate is
properly chosen.

Cheers,

Joshua

Sam ***@***.***> 于2021年4月6日周二 上午3:56写道：
